README.md

```diff
@@ -4,27 +4,42 @@
 
 ## What is Minishift?
 
-Minishift is a tool that makes it easy to run OpenShift locally. Minishift runs a single-node OpenShift cluster inside a VM on your laptop for users looking to try out OpenShift or develop with it day-to-day.
+Minishift is a tool that makes it easy to run OpenShift locally. Minishift runs a single-node OpenShift cluster inside a VM on your laptop if you want to try out OpenShift or develop with it day-to-day.
 
 ### Features
 
-* Minishift packages and configures a Linux VM, Docker and all OpenShift components, optimized for local development
+* Minishift packages and configures a Linux VM, Docker, and all OpenShift components, optimized for local development.
 * Minishift supports OpenShift features such as:
   * DNS
   * NodePorts
   * ConfigMaps and Secrets
   * Console
 
+### Design
+
+Minishift uses [libmachine](https://github.com/docker/machine/tree/master/libmachine) for provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) for running the cluster.
+
+## Known Issues
+
```

Comment:
+1 for pushing it down. Usually "known issues" is somewhere at the bottom. I want to primarily know what it is, how to install it and a quickstart.